packages/js/src/server/integrations/uncaught_exception.ts

```diff
@@ -14,27 +15,53 @@ function removeAwsLambdaListener() {
   removeAwsDefaultUncaughtExceptionListener()
 }
 
+/**
+ * If there are no other uncaughtException listeners,
+ * we want to report the exception to Honeybadger and
+ * mimic the default behavior of NodeJs,
+ * which is to exit the process with code 1
+ */
+function hasUncaughtExceptionListeners() {
+  return process.listeners('uncaughtException').length !== 0
+}
+
 export default function (): Types.Plugin {
   return {
     load: (client: typeof Client) => {
-      if (!client.config.enableUncaught) { return }
+      if (!client.config.enableUncaught) {
+        return
+      }
 
       removeAwsLambdaListener()
 
+      const hasOtherListeners = hasUncaughtExceptionListeners();
```

Comment:
Isn't there a slight bug here? If you only check for other listeners when this module is loaded, what happens if a listener is registered after ours? I think there `hasUncaughtExceptionListeners` should be called on demand. In which case, it should check if the length > 1 and probably be renamed to `hasOtherUncaughtExceptionListeners`.

Reply:
True. I would expect that this kind of setup happens quite early in any app. But you are right, I will improve! Thanks!